lib/generator.js

```diff
@@ -100,6 +101,7 @@
         options = options || {};
         self._options = options;
         self._config = options.config || {};
+        options.logger = _logger;
```

Comment:
Oh, I see where it is used in `photoshop.js` ... but I maintain that this is weird and bucks the trend of passing logger as a separate argument.  I think.